### PR TITLE
Better styling on the landing page

### PR DESCRIPTION
**Pull request by nchevobbe**
We simplify the rendered markup and some of the logic of the render function.
This allows us to also simplify the CSS, by mostly using the capabilities of
flexbox to have the same UI as we had before (no absolute positionning).
We also add some custom properties so the styling is more consistent.

### CODE REVIEW

File: packages/devtools-launchpad/src/components/LandingPage.js [hunk 1]
```diff
@@ -65,49 +65,41 @@ const LandingPage = React.createClass({
...
-      firstTimeMessage("Node", "node")
-    );
-  },
-
   renderPanel() {
-    const panels = {
-      Firefox: this.renderFirefoxPanel,
-      Chrome: this.renderChromePanel,
-      Node: this.renderNodePanel
+    const configMap = {
```
Comment (jasonLaster) [inline packages/devtools-launchpad/src/components/LandingPage.js]: lets move the configMap to the module context

File: packages/devtools-launchpad/src/components/LandingPage.js [hunk 2]
```diff
@@ -122,7 +114,7 @@ const LandingPage = React.createClass({
       connections.push("Chrome", "Node");
     }
 
-    return dom.div(
+    return dom.aside(
```
Comment (jasonLaster) [inline packages/devtools-launchpad/src/components/LandingPage.js]: 👍

File: packages/devtools-launchpad/src/components/LandingPage.js (hunk 1 above)
Comment (nchevobbe) [inline packages/devtools-launchpad/src/components/LandingPage.js]: Sure

**Comment by nchevobbe** [comment]
Could you add me as a contributor on the project so I can merge PRs when you're not around ? Thanks !

File: packages/devtools-launchpad/src/components/LandingPage.js (hunk 1 above)
Comment (nchevobbe) [inline packages/devtools-launchpad/src/components/LandingPage.js]: @jasonLaster I forgot to do this. I'll put it in a PR with other some small improvements. 
What do you mean by module context here ? this file https://github.com/devtools-html/devtools-core/blob/master/packages/devtools-launchpad/index.js ?

**Comment by jasonLaster** [comment]
Hmm not sure

**Pull request merged**